docs/en/operation/User-CLI.md

```diff
@@ -842,6 +842,12 @@ Later jobs must wait until some earlier jobs to finish. The default value is `30
 A lower value means slower execution but also being nicer to the other users of the job service.
 * `--host-file <host-file>`: Specifies a file contains worker hosts to load target data, each line has a worker host.
 * `--hosts`: Specifies a list of worker hosts separated by comma to load target data.
+* `--excluded-host-file <host-file>`: Specifies a file contains worker hosts which shouldn't load target data, each line has a worker host.
+* `--excluded-hosts`: Specifies a list of worker hosts separated by comma which shouldn't load target data.
+* `--locality-file <locality-file>`: Specifies a file contains worker locality to load target data, each line has a locality.
+* `--locality`: Specifies a list of worker locality separated by comma to load target data.
+* `--excluded-locality-file <locality-file>`: Specifies a file contains worker locality which shouldn't load target data, each line has a worker locality.
+* `--excluded-locality`: Specifies a list of worker locality separated by comma which shouldn't load target data.
 * `--index`: Specifies a file that lists all files to be loaded
 
 ```console
```

Comment:
(nit)Overall looks good to me, do you mind also add one example here with new options?

Reply:
distributedLoad --replication 2 --active-jobs 2000 --excluded-hosts <host > --excluded-locality <workerTierIdentity > /data/today